test/controllers/users_controller_test.rb

```diff
@@ -2,4 +2,84 @@
 
 describe UsersController do
 
+  describe "index" do
+
+    before do
+
+      @user_d = users(:dan)
+      @user_k = users(:kari)
+      @user_1 = users(:user_1)
+      @user_2 = users(:user_2)
+      @user_3 = users(:user_3)
+      @user_4 = users(:user_4)
+
+      @vote_1 = votes(:one)
+      @vote_2 = votes(:two)
+      @vote_3 = votes(:three)
+
+      @vote_1.destroy
+      @vote_2.destroy
+      @vote_3.destroy
+
+    end
+
+    it "succeeds when there is one user" do
```

Comment:
You should probably group this together for when the user is logged in an repeat the tests when they're not logged in.